.eslintrc.js

```diff
@@ -149,6 +149,8 @@ module.exports = {
     'rulesdir/prettier-comments': 'error',
     // Enforces consistent file extension
     'rulesdir/extensions': 'error',
+    // Enforces license headers on files (.ts,.mjs,.cjs)
```

Comment:
The files affected are controlled by EsLint so we can skip them.
```suggestion
    // Enforces license headers on files
```